packages/grafana-ui/src/types/icon.ts

```diff
@@ -189,7 +190,7 @@ export const getAvailableIcons = () =>
 
 type BrandIconNames = 'google' | 'microsoft' | 'github' | 'gitlab' | 'okta';
 
-export type IconName = ReturnType<typeof getAvailableIcons>[number] | BrandIconNames;
+export type IconName = ReturnType<typeof getAvailableIcons>[number] | BrandIconNames | AllUnicons;
```

Comment:
Will this type match only the icons that are available when the grafana/ui package is compiled and published to NPM (what is declared above in `getAvailableIcons`)  or will it contain 1000s which won't be available to a consumer outside of Grafana? 🤔

Reply:
>Will this type match only the icons that are available when the grafana/ui package is compiled and published to NPM
yes

> will it contain 1000s which won't be available to a consumer outside of Grafana?
also yes.

the type `AllUnicons` is generated and it contains all the icons present in `public/img/icons/unicons` at the time grafana/ui was built. So it will indeed include icons that are not included.